requirements/base.txt

```diff
@@ -1,5 +1,5 @@
-boto==2.22.1                        # MIT
 Django==1.8.14 						# BSD License
+boto==2.42.0                        # MIT
```

Comment:
We recently upgraded to Django 1.9 along with some of these third-party packages, so this will conflict, but master should be running on the same version of boto now.